dynabuffers-python/tests/usecase/Schema22Test.py

```diff
@@ -0,0 +1,32 @@
+import os
+import unittest
+
+from antlr4 import FileStream
+
+from dynabuffers.Dynabuffers import Dynabuffers
+
+
+class Schema22(unittest.TestCase):
+    root_dir = os.path.dirname(os.path.realpath(__file__))
+
+    def setUp(self):
+        self.engine = Dynabuffers.parse(FileStream(self.root_dir + "/schema22.dbs"))
+
+    def test(self):
+        values = [
+            9223372036854775807,  # long
```

Comment:
Was passiert mit Werten die größer sind als ein long? Ich nehme an die werden beim Serialisieren truncated zu einem Long? Ich denke dass wir da einen Fehler werfen sollten, wenn eine Zahl serialisiert wird, die mehr als 8 Byte braucht. (Vielleicht passiert das eh schon)

Reply:
Ja, da kommt ein Error, das wird getestet im Schema10Test. Soll ich's in diesem Schema22Test auch extra testen?

Reply:
Asso nein, das passt dann schon, danke!